src/axom/slic/core/Logger.hpp

```diff
@@ -65,6 +65,13 @@ class Logger
    */
   void setLoggingMsgLevel(message::Level level);
 
+  /*!
+   * \brief Aborts and flushes on warning or error if corresponding AbortOnError
+   * or AbortOnWarningis set to true
+   * \param [in] level the logging level.
+   */
```

Comment:
Is this a collective function?
(looks like it since it calls `flushStreams`)

Should we mark it somewhere?